fwoa-tools/src/integration-tests/migrationUtils.test.ts

```diff
@@ -3,11 +3,14 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 import { getFhirClient, getFhirClientSMART } from '../migrationUtils';
+import * as dotenv from "dotenv";
+import path from "path";
+dotenv.config({ path: path.resolve(__dirname, './.env') });
 
 describe('MigrationUtils', () => {
   describe('getFhirClient', () => {
     test('client should be able to authenticate', async () => {
-      await expect(getFhirClient()).resolves.not.toThrowError();
+      await expect(getFhirClient()).resolves.toBeDefined();
```

Comment:
`not.toThrowError()` wasn't working properly so I updated the code based on this SO
https://stackoverflow.com/questions/65793240/testing-for-an-error-not-being-thrown-by-an-async-function